queries/jsonnet/locals.scm

```diff
@@ -0,0 +1,18 @@
+(parenthesis) @scope
+(anonymous_function) @scope
+(object) @scope
+(field) @scope
+(local_bind) @scope
+
+(field
+  function: (fieldname (id)  @definition.function)
+  (#set! "definition.function.scope" "parent")
+)
```

Comment:
I feel like you will be pinged by ObserverOfTime to fix the formatting to be lisp-like, so I will ping you first :D

Reply:
Any `fmt` tools you can recommend?

Reply:
no `fmt` tools afaik, mostly just don't let `)` be on new line, rather put all the trailing on the final node

```
; not like this
...
  bar
)))
; but rather like this
 bar)))
```

Reply:
done

Reply:
This is not the only place. I only pointed out one example. Please do so for all files. I can see some in `highlights.scm`, for example

Reply:
@Duologic

Reply:
There will be one soon, though, so not necessary to overdo it. Just avoid hanging parens.

Reply:
I've checked and fixed, hopefully I haven't missed any.